src/components/__tests__/EmailValidationForm-test.tsx

```diff
@@ -12,25 +13,8 @@ describe("EmailValidationForm", () => {
   let fetchLibrary: Sinon.SinonStub;
 
   beforeEach(() => {
-    let library = {
-      uuid: "UUID1",
-      basic_info: {
-        "name": "Test Library 1",
-        "short_name": "lib1",
-        "description": undefined
-      },
-      urls_and_contact: {
-        "authentication_url": "auth1",
-        "contact_email": "email1",
-        "opds_url": "opds1",
-        "web_url": "web1",
-        "validated": "Not validated"
-      },
-      stages: {
-        "library_stage": "production",
-        "registry_stage": "testing"
-      }
-    };
+    const library = {...testLibrary1, ...{urls_and_contact: {...testLibrary1.urls_and_contact, ...{validated: "Not validated"}}}};
```

Comment:
You don't need the spread operator for `{urls_and_contact: ...` or `validated: "Not validated"` but the rest are okay. So it would be something like
```
const library = {...testLibrary1, urls_and_contact: { ...testLibrary1.urls_and_contact, validated: "Not validated" }};
```